civis/base.py

```diff
@@ -1,7 +1,10 @@
+from __future__ import absolute_import
```

Comment:
Is it worth including `division` with all of the `__future__` imports? I don't know if we're dividing anything anywhere, but that can introduce easy-to-overlook bugs.